src/ldp/http/OriginalUrlExtractor.ts

```diff
@@ -21,17 +21,17 @@ export class OriginalUrlExtractor extends TargetExtractor {
       throw new Error('Missing URL');
     }
 
-    // Extract host and protocol (possibly overridden by the Forwarded header)
-    let { host } = headers;
+    // Extract host and protocol (possibly overridden by the Forwarded/X-Forwarded-* header)
+    let { host }: { host?: string } = headers;
```

Comment:
Is this needed?

Reply:
Doesn't seem to be. I ran into an assignment warning at some point and added that as a result. However, that doesn't seem to be an issue with the current iteration.